t/live-https.t

```diff
@@ -0,0 +1,74 @@
+BEGIN {
+    unless ( -f "t/LIVE_TESTS" || -f "LIVE_TESTS" ) {
+        print "1..0 # SKIP Live tests disabled; pass --live-tests to Makefile.PL to enable\n";
+        exit;
+    }
+    eval {
```

Comment:
Could we use `Test::RequiresInternet` here? That would make the code more succinct.